interface/iface.go

```diff
@@ -1,19 +1,46 @@
 package zserio
 
-import "github.com/icza/bitio"
+import "io"
+
+type Reader interface {
+	io.Reader
+	io.ByteReader
+
+	ReadBits(n uint8) (uint64, error)
+	ReadBool() (bool, error)
```

Comment:
Should we have a separate `BitReader` interface, and make `Reader` a superset of `Reader`, `ByteReader` and `BitReader` ? Not sure how much value that would add, but at least it would be consistent with Reader/ByteReader.

Reply:
Done (both, the extra interface and the move).